src/foam/comics/DAOControllerView.js

```diff
@@ -145,6 +147,33 @@ foam.CLASS({
   methods: [
     function initE() {
       var self = this;
+      var summaryViewParent;
+
+      var reciprocalSearch =  foam.u2.ViewSpec.createView({
+        class: 'foam.u2.view.ReciprocalSearch',
+        data$: this.data.predicate$
+      },  {}, self, self.__subContext__.createSubContext({ memento: this.memento }));
+
+      var searchView = foam.u2.ViewSpec.createView({
+        class: 'foam.u2.view.SimpleSearch',
+        data$: this.data.predicate$
+      },  {}, self, reciprocalSearch.__subContext__);
+
+      if (this.data.searchMode === this.SearchMode.FULL) {
+        summaryViewParent = reciprocalSearch;
+      }
+      if ( this.data.searchMode === this.SearchMode.SIMPLE ) {
+        summaryViewParent = searchView;
+      }
+
+      var summaryView = foam.u2.ViewSpec.createView(this.summaryView, {
+        data$: this.data.filteredDAO$,
+        multiSelectEnabled: !! this.data.relationship,
```

Comment:
Does the !! do anything? If it is a boolean property it will do this anyway. If it isn't, it should be.

Reply:
I think it creates boolean from `this.data.relationship`